routes/admin.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');// pass passport for configuration
+const router = express.Router();
+const { isLoggedIn, isAdmin } = require('./utils/middleware') ;
+const user = require('./model/usermodel');
```

Comment:
@SparshKat Correct the path in lines 3 and 4. Also, paths will be changed in routes.js as it is moved to a new folder.

Reply:
Done!

Reply:
Required in routes.js too. Also, squash your commits.